libraries/AP_Baro/AP_Baro_HIL.cpp

```diff
@@ -40,6 +40,45 @@ void AP_Baro::SimpleAtmosphere(
     sigma = delta/theta;
 }
 
+void AP_Baro::SimpleUnderWaterAtmosphere(
+	float alt,            // depth, km.
+	float& rho,           // density/sea-level
+	float& delta,         // pressure/sea-level standard pressure
+	float& theta)         // temperature/sea-level standard temperature
+{
+    // Values and equations based on:
+    // https://en.wikipedia.org/wiki/Standard_sea_level
+    float seaDensity = 1.024f;      // g/cm3
+    float pAC = 0.004f;           // pycnocline angular coefficient
+    float maxSeaDensity = 1.028f; // g/cm3
+
+    // From: https://www.windows2universe.org/earth/Water/density.html
+    rho = seaDensity;
+    if(alt < 1.0f) {
+        // inside pycnocline
+        rho += pAC*alt;
+    } else {
+        rho += maxSeaDensity;
+    }
+    rho = rho/seaDensity;
+
+    // From: https://www.grc.nasa.gov/www/k-12/WindTunnel/Activities/fluid_pressure.html
+    // \f$P = \rho (kg) \cdot gravity (m/s2) \cdot depth (m)\f$
+    // \f$P_{atmosphere} = 101.325 kPa\f$
+    // \f$P_{total} = P_{atmosphere} + P_{fluid}\f$
+    float pAtm = 101.325e3;
```

Comment:
Just put 101325 this is misleading with units on first glance. Also, make the units in the code and the comments match.